### ISSUE TITLE
What about future of this project?

### ISSUE THREAD

**Issue by jroszkiewicz** [kryptco/kr#327]
You can see that the development of this project has been abandoned, so is there any fork planned?

**Comment by Oujiii** [comment]
The dev has sold the company and the replacement is this one: https://github.com/akamai/akr

**Comment by saadfarooq** [comment]
And that project is dead too now. No updates for a while.

**Comment by dominictayloruk** [comment]
Have a look at https://github.com/chiff-app

**Comment by jroszkiewicz** [comment]
> Have a look at https://github.com/chiff-app

Year without updates. Its look dead :(

**Comment by dominictayloruk** [comment]
Any other suggestions? 
Akami is garbage, krypton chrome extension is flagged as 'This extension violates the Chrome Web Store policy' and keeps getting deactivated in chrome on startup which is annoying.

**Comment by erwin** [comment]
> Any other suggestions?

It's not a 1:1 replacement for `kr`, but I've been using BitWarden and it's the best open source password management solution I've found. The UI is solid.

It's very easy to self host BitWarden by using VaultWarden instead of the very bloated/clunky official version.
https://github.com/dani-garcia/vaultwarden

The browser plugin for Firefox and Chrome based browsers gives you a place to switch to your self hosted BitWarden instance.

The mobile app also lets you point to your self hosted BitWarden instance.

**Comment by Oujiii** [comment]
@erwin This is not a 1:1 neither a 10000:1 solution. Bitwarden (in this case, vaultwarden) is awesome, but it is nowhere near what kr offered.

**Comment by erwin** [comment]
@Oujiii I agree that I would also like something that is more `ssh` focused, but for now I'm just stuck with `ssh-agent` again...

Instead of downvoting a useful reply, perhaps supply your useful information.

**Comment by saadfarooq** [comment]
I've moved to [secretive](https://github.com/maxgoedjen/secretive) since. My company also provides 1Password and their SSH produce seems pretty good as well.

Neither provide GPG though.

And my other complaint, which none of the products have solved so far, is that I need to use different keys for different repos and the best that these products seem to do is give you prompts for all the keys in order for you to decide which to use.

**Comment by dominictayloruk** [comment]
My following take on the suggested alternatives;
- **Bitwarden** - Password manager not MFA _(I use Bitwarden!)_
- **KeePass** - Same features as Bitwarden
- **Secretive** - Offers some functionality similar to kr but requires a secure enclave on Mac _(Phone not the security key)_
- **Apple Passkeys** - Offer the Webauthn of krypton but only able to use in Safari on iOS16/MacOS Ventura _(No chrome/Firefox)_
- **Chiff** - Looks like the best contender for feature parity with krypton but hasn't been updated in a while as @jroszkiewicz said

**Comment by JamesMcMahon** [comment]
There is also https://developer.1password.com/docs/ssh/agent.

Haven't tried it yet, but seems comparable since it is also an SSH agent. Downside is the dependency on 1Password.

**Comment by nikhilty** [comment]
This project has been moved to https://github.com/akamai/akr and is under active development.
Let me know if there is any issue i can address over there.
